pypeit/par/pypeitpar.py

```diff
@@ -1476,6 +1539,7 @@ def validate(self):
 
 
 
+
 class CubePar(ParSet):
     """
     The parameter set used to hold arguments for functionality relevant
```

Comment:
See below, there is a call to `relative_weights` here that should be changed to `weight_method`... Although, I'm not sure if it's possible to call `Coadd1DPar.valid_weight_methods()`, but that would be ideal if it's possible.

The alternative is to remove both `relative_weights` and `weight_method` from the `CubePar()` class and only call the CoAdd1D method. This may be a little messy though... probably needs a little extra thought.

Reply:
Okay I think I fixed it.